tests/test_task_collection.py

```diff
@@ -89,15 +94,41 @@ def __restore_perceval_cache(self):
 
     def test_initialization(self):
         """Test whether attributes are initializated"""
+
         config = Config(CONF_FILE)
         backend_section = GIT_BACKEND_SECTION
         task = TaskRawDataCollection(config, backend_section=backend_section)
 
         self.assertEqual(task.config, config)
         self.assertEqual(task.backend_section, backend_section)
 
+    def test_backend_params(self):
+        """Test whether the backend parameters are initializated"""
+
+        config = Config(CONF_FILE)
+        backend_section = GITHUB_BACKEND_SECTION
+        task = TaskRawDataCollection(config, backend_section=backend_section)
+        params = task._compose_perceval_params(GITHUB_BACKEND_SECTION, GITHUB_REPO)
+
+        expected_params = [
+            'grimoirelab',
+            'perceval',
+            '--api-token',
+            'XXXXX',
+            '--sleep-time',
+            '300',
+            '--sleep-for-rate',
+            '--no-cache'
+        ]
+
+        self.assertEqual(len(params), len(expected_params))
+
+        for p in params:
+            self.assertTrue(p in expected_params)
+
```

Comment:
Nice test. Maybe you can rework this PR to include only the code related to this test!